appveyor.yml

```diff
@@ -6,12 +6,22 @@ before_build:
   - nuget restore nunit.sln
 
 build_script: 
-  - msbuild nunit.sln /p:Configuration=Release /t:Rebuild
+  - ps: |
+      .\appveyor.ps1
+      if ($lastexitcode -ne 0){ exit $lastexitcode }
   
-test_script:
-  - msbuild NUnit.proj /t:TestAll /p:Configuration=Release /p:ResultFormat=nunit2 /p:ContinueOnFailure=ErrorAndContinue
+#test_script:
+#  - msbuild NUnit.proj /t:TestAll /p:Configuration=Release /p:ResultFormat=nunit2 /p:ContinueOnFailure=ErrorAndContinue
 #  - ps: $wc = New-Object 'System.Net.WebClient'
 #  - ps: $dir = "bin\Release\Results"
 #  - ps: $uri = "https://ci.appveyor.com/api/testresults/nunit/$($env:APPVEYOR_JOB_ID)"
 #  - ps: foreach($item in (dir $dir "*.xml")) { $wc.UploadFile($uri, $item.FullName) }
   
+artifacts:
```

Comment:
Looks like a possibility but doesn't change the fact that anyone can fork
us and change appveyor.yml

On Mon, Oct 26, 2015 at 8:44 AM, Rob Prouse <notifications@github.com>
wrote:

> In appveyor.yml
> <https://github.com/nunit/nunit/pull/932#discussion_r43010247>:
>
> >  #  - ps: $wc = New-Object 'System.Net.WebClient'
> >  #  - ps: $dir = "bin\Release\Results"
> >  #  - ps: $uri = "https://ci.appveyor.com/api/testresults/nunit/$($env:APPVEYOR_JOB_ID)"
> >  #  - ps: foreach($item in (dir $dir "*.xml")) { $wc.UploadFile($uri, $item.FullName) }
> >
> > +artifacts:
>
> Does on branch work with artifacts, or just deployments? If not, we could
> use a conditional build configuration,
> http://www.appveyor.com/docs/branches#conditional-build-configuration
>
> on:
>       branch: master
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nunit/nunit/pull/932/files#r43010247>.
>